pkgs/applications/audio/tamgamp.lv2/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub,  zita-resampler, lv2 }:
+
+stdenv.mkDerivation rec {
+  pname = "tamgamp.lv2";
+  version = "unstable-2020-05-14";
+
+  src = fetchFromGitHub {
+    owner = "sadko4u";
+    repo = pname;
+    rev = "590ced0a1da96ca481a1a719eebdb17f3af472e4";
+    sha256 = "0yqdhbxsx2h1w1wwf7db7a7w9dkqmnd9c2visiznmcl9chinyhiq";
+  };
+
+  buildInputs = [
+    zita-resampler lv2
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/sadko4u/tamgamp.lv2;
```

Comment:
as per [RFC0045](https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md) URLs should be quoted